packages/compass-e2e-tests/tests/connection.test.ts

```diff
@@ -543,96 +752,15 @@ describe('Connection screen', function () {
-    if (!hasAtlasEnvironmentVariables()) {
-      return this.skip();
-    }
-
-    await browser.connectWithConnectionString(
-      process.env.E2E_TESTS_ATLAS_READWRITEANY_STRING ?? ''
-    );
-    const result = await browser.shellEval(
-      'db.runCommand({ connectionStatus: 1 })',
-      true
-    );
-    assertNotError(result);
-    expect(result).to.have.property('ok', 1);
-
-    await assertCanReadData(browser, 'compass_e2e', 'companies_info');
-  });
-
-  it('can connect with readAnyDatabase builtin role', async function () {
-    if (!hasAtlasEnvironmentVariables()) {
-      return this.skip();
-    }
-
-    await browser.connectWithConnectionString(
-      process.env.E2E_TESTS_ATLAS_READANYDATABASE_STRING ?? ''
-    );
-    const result = await browser.shellEval(
-      'db.runCommand({ connectionStatus: 1 })',
-      true
-    );
-    assertNotError(result);
-    expect(result).to.have.property('ok', 1);
-
-    await assertCanReadData(browser, 'compass_e2e', 'companies_info');
-    await assertCannotInsertData(browser, 'compass_e2e', 'companies_info');
-    await assertCannotCreateDb(browser, 'new-db', 'new-collection');
-    await assertCannotCreateCollection(
-      browser,
-      'compass_e2e',
-      'new-collection'
-    );
-  });
-
-  it('can connect with custom role', async function () {
-    if (!hasAtlasEnvironmentVariables()) {
-      return this.skip();
-    }
-
-    await browser.connectWithConnectionString(
-      process.env.E2E_TESTS_ATLAS_CUSTOMROLE_STRING ?? ''
-    );
-    const result = await browser.shellEval(
-      'db.runCommand({ connectionStatus: 1 })',
-      true
-    );
-    assertNotError(result);
-    expect(result).to.have.property('ok', 1);
-
-    await assertCanReadData(browser, 'test', 'users');
-    await assertCannotCreateDb(browser, 'new-db', 'new-collection');
-    await assertCannotCreateCollection(browser, 'test', 'new-collection');
-  });
-
-  it('can connect with read one collection specific permission', async function () {
-    if (!hasAtlasEnvironmentVariables()) {
-      return this.skip();
-    }
-
-    await browser.connectWithConnectionString(
-      process.env.E2E_TESTS_ATLAS_SPECIFICPERMISSION_STRING ?? ''
-    );
-    const result = await browser.shellEval(
-      'db.runCommand({ connectionStatus: 1 })',
-      true
-    );
-    assertNotError(result);
-    expect(result).to.have.property('ok', 1);
-
-    await assertCanReadData(browser, 'test', 'users');
-    await assertCannotInsertData(browser, 'test', 'users');
-    await assertCannotCreateDb(browser, 'new-db', 'new-collection');
-    await assertCannotCreateCollection(browser, 'test', 'new-collection');
-  });
 });
 
 // eslint-disable-next-line mocha/max-top-level-suites
 describe('SRV connectivity', function () {
   before(function () {
-    skipForWeb(this, 'not applicable to compass-web');
+    skipForWeb(
+      this,
+      'we do not support the either connectionStatus param in compass-web yet'
```

Comment:
See https://github.com/mongodb-js/compass/blob/4eb3befc9c07e7e115aa7412cac7f8d6898847ef/packages/compass-e2e-tests/helpers/commands/wait-for-connection-result.ts#L12

Reply:
okay, but even if we did support `connectionStatus` in compass-web, this would still not be applicable to it?